src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterType,
+  handleFilterType: (filter: FilterType) => void,
+  handleCompletedTodos: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  handleFilterType,
+  handleCompletedTodos,
+}) => {
+  const todosLeft = useMemo(() => {
+    return todos.filter(todo => !todo.completed).length;
+  }, [todos]);
+  const todosCompleted = useMemo(() => {
```

Comment:
I think for enough just one completed todo to be present, to show the button, you can use some